.flowconfig

```diff
@@ -24,7 +24,6 @@
 [libs]
 node_modules/react-native/Libraries/react-native/react-native-interface.js
 node_modules/react-native/flow/
-node_modules/react-native/flow-github/
```

Comment:
Removed to fix this notice:

![Screen Shot 2019-03-13 at 17 57 04](https://user-images.githubusercontent.com/73652/54349415-1cb6b580-4643-11e9-9e7c-5599969d0b6b.png)

Reply:
Can we also have a look at the `.flowconfig` of RN at 0.59.0? That might contain some new configuration that we should also add.

Reply:
I was going to do that, but if the config we have is already working for us, what is there to be gained by changing it?

Reply:
We used that mentality previously, only to find out that we were missing out on a bunch of flow functionality. 

In other words, when we aligned our version of flow to RN's version and subsequently aligned the `.flowconfig`, we found errors we were otherwise oblivious to. 

Who knows, perhaps https://github.com/Skyscanner/backpack-react-native/pull/77#discussion_r265600509 goes away when we align the config?

Reply:
That's a fair point! Let me investigate 🤓

Reply:
I've updated it based on RN's latest configs, no changes it seems (including https://github.com/Skyscanner/backpack-react-native/pull/77#discussion_r265600509 unfortunately) but as you say, good to be aligned with the latest and greatest.